pkg/intermediate/aggregate.go

```diff
@@ -222,20 +223,22 @@ func getFlowKeyFromRecord(record entities.Record) (*FlowKey, error) {
 			if !ok {
 				return nil, fmt.Errorf("%s is not in correct format", name)
 			}
-			isIPv4 = true
+
 			if strings.Contains(name, "source") {
+				isSrcFilled = true
 				flowKey.SourceAddress = addr.String()
 			} else {
+				isDstFilled = true
 				flowKey.DestinationAddress = addr.String()
 			}
+			if isSrcFilled && isDstFilled {
+				break checkFlowKey
+			}
 		case "sourceIPv6Address", "destinationIPv6Address":
-			element, exist := record.GetInfoElementWithValue(name)
-			if isIPv4 {
-				if exist {
-					klog.Warning("two ip version (ipv4 & ipv6) is not supported at the same time in Flow Key.")
-				}
+			if (isSrcFilled && strings.Contains(name, "source")) || (isDstFilled && strings.Contains(name, "destination")) {
```

Comment:
Do we want to print warning if both (src ipv4 and src ipv6) or (dst ipv4 and dst ipv6) are set?

Reply:
Sure. Updated